Doc/library/stdtypes.rst

```diff
@@ -1549,6 +1549,39 @@ expression support in the :mod:`re` module).
    interpreted as in slice notation.
 
 
+.. method:: str.removeprefix(prefix, /)
+
+   Return a copy of the string with the given prefix removed, if present. ::
+
+      >>> 'BarFooBaz'.removeprefix('Bar')
+      'FooBaz'
+      >>> 'BarFooBaz'.removeprefix('Baz')
+      'BarFooBaz'
+
+   The expression ``s.removeprefix(pre)`` is roughly equivalent to
+   ``s[len(pre):] if s.startswith(pre) else s``.
+   Unlike :meth:`~str.startswith`, only one prefix can be passed
+   at a time.
```

Comment:
I don't think that it's worth it to mention that. Same remark for removesuffix() doc.